src/apm/php.go

```diff
@@ -133,10 +148,12 @@ func (i *PhpInjector) Inject(ctx context.Context, inst v1alpha2.Instrumentation,
 				Name:      volumeName,
 				MountPath: "/newrelic-instrumentation",
 			}},
-		})
+		}
+		initContainer = i.injectNewrelicLicenseKeyIntoContainer(initContainer, inst.Spec.LicenseKeySecret)
```

Comment:
Does this solve the license key injection into the init container issue?

Reply:
Yes.  Rather than injecting the api key into the main container like the other apm agents, this specifically injects into the initContainer